com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -312,7 +312,7 @@ public void Dispose()
             // Dispose all renderer features...
             for (int i = 0; i < m_RendererFeatures.Count; ++i)
             {
-                rendererFeatures[i].Dispose();
```

Comment:
This doesn't work properly with the Unity Object null check, should be replace with an actual `rendererFeatures[i] == null` check.